app/views/movies/show.html.erb

```diff
@@ -0,0 +1,19 @@
+<%= @movie.title %>
+<%= @movie.creation_year %>
+<%= @movie.genre %>
+<%= @movie.actor_age_avg %>
+
+<% @movie.actors_sorted.each do |actor| %>
+  <div id="actor-<%= actor.id %>">
+    <%= actor.name %>
+    <%= actor.age %>
+  </div>
+<% end %>
+
+<%= form_with url: "/movies/#{@movie.id}/actors", method: :post, data: { turbo: false } do |f| %>
+  <%= f.label :actor %>
+  <%= f.text_field :actor %>
+
+  <%= f.hidden_field :movie_id, value: @movie.id %>
+  <%= f.submit "Add Actor" %>
+<% end %>
```

Comment:
Yes! This form is perfect and follows restful conventions perfectly! You're using the correct verb (POST) and sending the data to the correct path, great job Sam!

The only thing I will say is that you do not need the hidden field for the movie ID since you are going to be getting the movie ID from the URL. This can be removed.